acceptance/cluster/localcluster.go

```diff
@@ -336,7 +335,6 @@ func (l *LocalCluster) createRoach(node *testNode, vols *Container, cmd ...strin
 
 	hostConfig := container.HostConfig{
 		PublishAllPorts: true,
-		NetworkMode:     container.NetworkMode(l.networkID),
```

Comment:
Ah, just missed that (I read the commit message, but wasn't reading the code properly).